test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -154,6 +147,11 @@ private function assertSameClassAttributes(CoreReflectionClass $original, Reflec
         $this->assertSameInterfaces($original, $stubbed);
 
         foreach ($original->getMethods() as $method) {
+            // Needs fix in JetBrains/phpstorm-stubs
```

Comment:
Can we make an issue out of this?

Reply:
https://github.com/JetBrains/phpstorm-stubs/pull/573
https://github.com/JetBrains/phpstorm-stubs/pull/574
https://github.com/JetBrains/phpstorm-stubs/pull/583
https://github.com/JetBrains/phpstorm-stubs/pull/587
https://github.com/JetBrains/phpstorm-stubs/pull/582
https://github.com/JetBrains/phpstorm-stubs/pull/584
https://github.com/JetBrains/phpstorm-stubs/pull/585

https://github.com/php/php-src/pull/4139

And I'm not done yet :)

Reply:
Ouch xD